github/ipld.yml

```diff
@@ -2,18 +2,13 @@
 
 members:
   admin:
+    # Admin permissions are intentionally minimal. 
+    # Permissoins are distributed across 3-4 separate organizations.
+    # One can request additional permissions using ipld/github-mgmt.
     - andyschwab-admin
     - aschmahmann
-    - daviddias
-    - dignifiedquire
     - galargh
-    - jbenet
-    - marten-seemann
-    - rvagg
```

Comment:
eh? I'd like to hear a justification for the list left in here

Reply:
I made this change (quickly).

I assume there is no questioning about the general push to reduce this given the power an "organization admin" has per https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization#permissions-for-organization-roles.

I assume the concern is more on why was @rvagg removed? :)

This was my doing of a proposal to pull one person from a few different organizations knowing additional people could be added necessary through github-mgmt and often times don't need for organizational ownership access.  I quickly pulled vmx for PLGO given he had just commented on the PR.  It was an oversight not to include your Rod.  I personally have no concerns with adding you back in as an owner/admin.  I'll put that in as a suggestion now.

(terraform members.admin is a github "org owner")

Reply:
I add @rvagg back as admin.  I'm also seeing in the process I had fully removed him as a member as well.  That certainly wasn't intentional!  Anyways, all fixed now.

Reply:
I agree with the push to simplify this, and even the ideal of trimming the admin permissions right down to near zero (although zero would imply a great deal of faith in the tooling and it being future proof); I was more wondering if there's context that I missed in that one of the only two people who contribute to, and maintain this org on a regular basis were removed from the list, leaving three who don't (nucleation uncertainty opens up all sorts of interesting interpretations).